week1/scary/kirikaza/.gitignore

```diff
@@ -0,0 +1,2 @@
+/*.hp
```

Comment:
I wanted to ensure `getContents` with `lines` don't load all lines into memory. I've found a way to compile with `--profile`, run to generate *.hp and then use hp2ps to generate *.ps.